src/main/java/org/zalando/nakadi/service/EventPublisher.java

```diff
@@ -145,6 +151,11 @@ private void validateSchema(final JSONObject event, final EventType eventType) t
         }
     }
 
+    private void validateEventSize(final BatchItem item) throws EventValidationException {
+        if (item.getEvent().toString().getBytes().length > nakadiSettings.getEventMaxBytes())
```

Comment:
Never use `getBytes()` without setting the default locale. Please always pass https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html#UTF_8 object into that function.

Actually we have to check that we do not use cases where this stupid `getByte()` is done.